src/pages/Profile/ProfileMain/Informaion/Informaion.js

```diff
@@ -0,0 +1,82 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import CareerCard from './CareerCard';
+import EducationCard from './EducationCard';
+import styled from 'styled-components';
+
+export default function InformaionCRUD(props) {
+  const { title, cardData } = props;
+
+  return (
+    <StyledInformation>
+      {title.includes('경력') ? (
```

Comment:
```suggestion
      {title.includes('경력') && (
```

`&&` 로 묶어주는 대신 삼항연산자로 `null` 처리하신 특별한 이유가 있을까요?
둘의 차이점을 깊게 고민해본 적은 아직 없어서요ㅎㅎ

Reply:
이거슨.. 제가 아직 단축평가가 완전히 익숙하지 않아서 생각나는 삼항을 먼저 써버렸습니다 ㅎㅎㅎㅎ... 앞으로는 삼항보다 &&을 쓸 예정입니다~ㅎㅎㅎ